include/itkCommandExhaustiveLog.h

```diff
@@ -0,0 +1,132 @@
+/*=========================================================================
+ *
+ *  Copyright NumFOCUS
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0.txt
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ *
+ *=========================================================================*/
+#ifndef itkCommandExhaustiveLog_h
+#define itkCommandExhaustiveLog_h
+
+#include "itkMacro.h"
+#include "itkCommand.h"
+#include "itkImage.h"
+#include "itkExhaustiveOptimizerv4.h"
+
+namespace itk
+{
+    /**
+     *\class CommandExhaustiveLog
+     *  \brief Command class to collect data from itk::ExhaustiveOptimizerv4 class.
```

Comment:
The introductory paragraph of the example notebook was more useful for understanding the purpose of this class than the documentation here. Perhaps update the docs here?

Reply:
Perhaps: `Monitors ExhaustiveOptimizerv4 and provides the search space as an image.`

Reply:
Thanks for the feedback @dzenanz ! I added the brief you suggested and expanded the inline summary.